src/cli/watcher.js

```diff
@@ -0,0 +1,84 @@
+const chokidar = require('chokidar')
+const path = require('path')
+
+module.exports = class Watcher {
+  constructor(options) {
+    const { onReady, onTrigger, onCollectFiles, onError, onFilesUpdated } = options
```

Comment:
Is`onFilesUpdated` needed? It appears to be unused.

Reply:
Indeed it isn't used. I've removed it.